src/visibility.js

```diff
@@ -129,9 +135,13 @@ class Visibility extends BasePlugin {
   _startFloating() {
     Utils.Dom.addClassName(this._floatingContainer, FLOATING_ACTIVE_CLASS);
     Utils.Dom.addClassName(this._floatingPoster, FLOATING_POSTER_CLASS_SHOW);
-    Utils.Dom.setStyle(this._floatingContainer, 'height', this.config.height + 'px');
+    Utils.Dom.setStyle(this._floatingContainer, 'height', `${this._floatingContainerHeight}px`);
     Utils.Dom.setStyle(this._floatingContainer, 'width', this.config.width + 'px');
     Utils.Dom.setStyle(this._floatingContainer, 'margin', `${this.config.marginY}px ${this.config.marginX}px`);
+    if (this.config.dismissible) {
+      const dismissibleContainerEl = Utils.Dom.getElementById(FLOATING_DISMISSIBLE_CONTAINER_ID);
+      this._floatingContainer.prepend(dismissibleContainerEl);
```

Comment:
why add / remove the element instead of just showing / hiding it ?

Reply:
actually it is playing with show/hide, I just had to move it into the `floatingContainer` because of the new design